docs/pages/integration-test/masonry.js

```diff
@@ -18,10 +19,19 @@ function booleanize(value: string): boolean {
   return Boolean(value);
 }
 
+// LazyHydrate doesn't like to be used without any props, so we have to add it conditionally
+function MaybeLazyHydrate({ children, ssrOnly }: {| children: Node, ssrOnly: boolean |}) {
+  if (ssrOnly) {
+    return <LazyHydrate ssrOnly>{children}</LazyHydrate>;
```

Comment:
huh, TIL: https://github.com/facebook/react/issues/10923#issuecomment-338715787

Reply:
Interesting…how would you see us using that here? (that thread definitely would've been useful to me in a previous job where I was using D3 with React)

Reply:
just added that because its what LazyHydrate does under the hood (was curious about how it handles it and i found that)